staging/src/k8s.io/kube-apiextensions-server/pkg/apiserver/apiserver.go

```diff
@@ -56,6 +56,17 @@ var (
 	registry             = registered.NewOrDie("")
 	Scheme               = runtime.NewScheme()
 	Codecs               = serializer.NewCodecFactory(Scheme)
+
+	// if you modify this, make sure you update the crEncoder
+	unversionedVersion = schema.GroupVersion{Group: "", Version: "v1"}
```

Comment:
> #46112

Commented in that pull.  I'm not sure enough to merge it over clayton's objection.